From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: [PATCH] USB: Fix GPL markings on usb core functions.
Patch-mainline: 2.6.16-rc3

tree baaa6598d79f94decad1d29857b5fcbfaea20d97
parent 7a21ef6fe902ac0ad53b45af6851ae5ec3a64299
author Greg KH <gregkh@suse.de> Mon, 06 Feb 2006 06:16:08 -0800
committer Linus Torvalds <torvalds@g5.osdl.org> Mon, 06 Feb 2006 06:20:14 -0800

[PATCH] USB: Fix GPL markings on usb core functions.

I thought we had fixed up all non-gpl USB drivers, and was wrong to do
this.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
Signed-off-by: Linus Torvalds <torvalds@osdl.org>


---
 drivers/usb/core/driver.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- gregkh-2.6.orig/drivers/usb/core/driver.c
+++ gregkh-2.6/drivers/usb/core/driver.c
@@ -378,7 +378,7 @@ const struct usb_device_id *usb_match_id
 
 	return NULL;
 }
-EXPORT_SYMBOL_GPL(usb_match_id);
+EXPORT_SYMBOL(usb_match_id);
 
 int usb_device_match(struct device *dev, struct device_driver *drv)
 {
@@ -446,7 +446,7 @@ int usb_register_driver(struct usb_drive
 
 	return retval;
 }
-EXPORT_SYMBOL_GPL(usb_register_driver);
+EXPORT_SYMBOL(usb_register_driver);
 
 /**
  * usb_deregister - unregister a USB driver
@@ -469,4 +469,4 @@ void usb_deregister(struct usb_driver *d
 
 	usbfs_update_special();
 }
-EXPORT_SYMBOL_GPL(usb_deregister);
+EXPORT_SYMBOL(usb_deregister);
